torch_xla/csrc/ops/flash_attention_forward.cpp

```diff
@@ -235,13 +235,25 @@ void custom_call_flash_attention_forward(cudaStream_t stream, void** buffers,
   // inference.
   const int num_m_blocks = (params.seqlen_q + 64 - 1) / 64;
   launch_params.num_splits = 0;
-  if (1 - params.p_dropout == 0.0f) {  // SplitKV is not implemented for dropout
+  if (params.p_dropout == 1.0f) {  // SplitKV is not implemented for dropout
     if (launch_params.num_splits < 1) {
       auto dprops = at::cuda::getCurrentDeviceProperties();
       launch_params.num_splits =
           num_splits_heuristic(params.b * params.h * num_m_blocks,
                                dprops->multiProcessorCount, num_n_blocks, 128);
     }
+    if (launch_params.num_splits > 1) {
+      at::Tensor softmax_lse_accum =
+          torch::empty({launch_params.num_splits, params.b, params.h,
+                        launch_params.seqlen_q},
+                       opts.dtype(at::kFloat));
+      at::Tensor out_accum =
```

Comment:
Will these two tensors be destructed once they leave the scope of the if statement?

Reply:
You're right, I've modified it.